LoRaEngine/modules/LoRaSimulator/LoRaSimulator/SimulatedDevice.cs

```diff
@@ -91,7 +91,7 @@ public byte[] GetUnconfirmedDataUpMessage()
             Array.Reverse(_payload);
             // 0 = uplink, 1 = downlink
             int direction = 0;
-            LoRaPayloadData standardData = new LoRaPayloadData(_mhbr, _devAddr, _FCtrl, _FCnt, _Fopts, _FPort, _payload, direction);
+            LoRaPayloadData standardData = new LoRaPayloadData((LoRaPayloadData.MType)_mhbr[0], _devAddr, _FCtrl, _FCnt, _Fopts, _FPort, _payload, direction);
```

Comment:
We chose to move away from _xxx notation, we should probably refactor this part of the solution

Reply:
I agree. My suggestion would be to integrate code analysis as warning. Then start fixing the warnings as we touch classes/methods.

What do you think?

Reply:
yes I did play around StyleCop roslyin analyzer... somtihng like 700 errors but we should start using it. Or do you have any other preference?

Reply:
On top of my head I know FxCop and StyleCop analysers. If you agree, I can start working on it after the CI is done.

Reply:
🎉